packages/muelu/doc/Tutorial/README.md

```diff
@@ -0,0 +1,37 @@
+[![Documentation Status](https://readthedocs.org/projects/first-steps/badge/?version=latest)](https://first-steps.readthedocs.io/en/latest/?badge=latest)
+
+# first_steps_readthedocs
+First steps with readthedocs
+
+## Tools to install
+
+1. `pip install sphinx`
+1. `pip3 install sphinx_rtd_theme`
+
+> Note: `pip` might have to be replaced by `pip3` for `python3`.
+
+## Setup documentation via `readthedocs`
+
+1. `cd <path/to/project>
+1. `mkdir docs`
+1. `cd docs`
+1. `sphinx-quickstart`
+
+Add option
+```
+master_doc = 'index'
+```
+to `conf.py` to point the readthedocs site generator to `index.rst`, when building and deploying the site.
```

Comment:
This can probably go away. I used this to explain to myself how to setup a read-the-docs project from scratch. Since we have it, this should not be needed anymore. 

We might replace this with a brief overview along the lines of "This directory contains all materials of the MueLu tutorial.", just to explain the purpose of these files.